src/interner.rs

```diff
@@ -204,6 +224,13 @@ where
             backend,
         } = self;
         let hash = make_hash(hasher, string.as_ref());
+        // this call requires hashbrown `raw` feature enabled
```

Comment:
please remove superflous comment

Reply:
Sure, this was more of a message to you than the real code comment.